migration/migration.go

```diff
@@ -254,6 +254,9 @@ func BootstrapWorkItemLinking(ctx context.Context, linkCatRepo *link.GormWorkIte
 	if err := createOrUpdateWorkItemLinkType(ctx, linkCatRepo, linkTypeRepo, link.SystemWorkItemLinkPlannerItemRelated, "One planner item or a subtype of it relates to another one.", link.TopologyNetwork, "relates to", "relates to", workitem.SystemPlannerItem, workitem.SystemPlannerItem, link.SystemWorkItemLinkCategorySystem); err != nil {
 		return err
 	}
+	if err := createOrUpdateWorkItemLinkType(ctx, linkCatRepo, linkTypeRepo, link.SystemWorkItemLinkEpicRelatedUserStory, "Epic includes user story and vice versa", link.TopologyNetwork, "includes", "is included in", workitem.SystemEpic, workitem.SystemUserStory, link.SystemWorkItemLinkCategorySystem); err != nil {
```

Comment:
@Ritsyy 
> "Epic includes user story and vice versa".
Actually, the way that you designed the link type, it is not "vice versa" because a user story cannot include an epic.